openlibrary/plugins/openlibrary/js/signup.js

```diff
@@ -163,7 +165,7 @@ export function initSignupForm() {
             validateUsername();
         } else if (id === 'password') {
             validatePassword();
-        } else {
+        } else if (id !== 'pd_request') {
             throw new Error('Input validation function not found');
         }
```

Comment:
```suggestion
        } else if (id === 'pd_request') {
            // TODO: ensure that a qualifying authority is selected
        } else {
            throw new Error('Input validation function not found');
        }
```
`pda` value must be validated before form is submitted.